### PR TITLE
Update gir / gir-files and update for `time_t`

### PR DESCRIPTION
**Pull request by sdroege**

### CODE REVIEW

File: gio/sys/src/lib.rs [hunk 1]
```diff
@@ -10391,11 +10394,11 @@ extern "C" {
     pub fn g_credentials_get_unix_pid(
         credentials: *mut GCredentials,
         error: *mut *mut glib::GError,
-    ) -> c_int;
+    ) -> pid_t;
```
Comment (sdroege) [inline gio/sys/src/lib.rs:10397]: All this API should only exist on UNIX platforms

```
error[E0412]: cannot find type `pid_t` in this scope
Error:      --> gio\sys\src\lib.rs:10397:10
      |
10397 |     ) -> pid_t;
      |          ^^^^^ not found in this scope

error[E0412]: cannot find type `uid_t` in this scope
Error:      --> gio\sys\src\lib.rs:10401:10
      |
10401 |     ) -> uid_t;
      |          ^^^^^ not found in this scope

error[E0412]: cannot find type `uid_t` in this scope
Error:      --> gio\sys\src\lib.rs:10414:14
      |
10414 |         uid: uid_t,
      |              ^^^^^ not found in this scope
```

File: glib/sys/Cargo.toml [hunk 2]
```diff
@@ -93,7 +94,10 @@ version = "2.76"
 version = "2.77"
 
 [package.metadata.system-deps.glib_2_0.v2_80]
-version = "2.79"
+version = "2.80"
```
Comment (sdroege) [inline glib/sys/Cargo.toml:97]:
```
  > PKG_CONFIG_ALLOW_SYSTEM_CFLAGS=1 C:\gtk-build\gtk\x64\release\bin\pkgconf.exe --libs --cflags glib-2.0 glib-2.0 >= 2.80

  The system library `glib-2.0` required by crate `glib-sys` was not found.
  The file `glib-2.0.pc` needs to be installed and the PKG_CONFIG_PATH environment variable must contain its parent directory.
  The PKG_CONFIG_PATH environment variable is not set.
```

It looks like Windows currently uses GLib 2.79.X? How do we update that @bilelmoussaoui ? :)

Comment (bilelmoussaoui) [reply glib/sys/Cargo.toml:97]: @pbor is the windows expert in this repo, i only know how to deal with msvc

Comment (sdroege) [reply glib/sys/Cargo.toml:97]: Maybe like this. Let's see

Comment (sdroege) [reply glib/sys/Cargo.toml:97]: Yes

**Comment by sdroege** [comment]
@bilelmoussaoui This would be ready now. Please check :)

**Review by bilelmoussaoui — APPROVED** [review]

**Pull request merged**